targets/drive/mobile/config.xml

```diff
@@ -48,6 +45,7 @@
     </platform>
     <engine name="ios" spec="4.5.4" />
     <platform name="ios">
+        <hook src="scripts/configure-share-extension.js" type="after_plugin_install" />
```

Comment:
There are a few things that need to be done in xcode after the plugin is installed, so i've scripted them.
However, the `after_prepare` hook runs *before* all the plugins are installed, and there's no `after_plugins` hook. So this script runs after *every* plugin install. That sucks, but keep reading.